### DISCUSSION TITLE
v6.10.2

### DISCUSSION THREAD

**Discussion by ssbarnea** [ansible/ansible-lint#2863] [General]
## Bugfixes

- Make git a runtime dependency (#2860) @ssbarnea
- Disable version check when running as a pre-commit hook (#2858) @ssbarnea
- Avoid checking var-names on non ansible files (#2856) @ssbarnea

<hr /><em>This discussion was created from the release <a href='https://github.com/ansible/ansible-lint/releases/tag/v6.10.2'>v6.10.2</a>.</em>

**Comment by bcdady** [comment]
Making git a runtime dependency results in a 'breaking' level change and so should be more than a patch version increment, right?
see also [#2859](https://github.com/ansible/ansible-lint/discussions/2859#discussioncomment-4576474)

**Reply by ssbarnea** [reply]
Is more in a grey area because git was already a runtime dependency, the difference is that now we no longer ignore its absence and we ensure that tool behaviour is consistent.
